src/commands/WeatherCommand.js

```diff
@@ -50,7 +50,7 @@ class WeatherCommand extends Command {
 
                 const weatherEmbedTomorrow = new Discord.RichEmbed()
                     .setColor(0xbadc58)
-                    .setTitle(Translation.translate("command.weather.tomorrow") + currentLocation, true)
+                    .setTitle(Translation.translate("command.weather.tomorrow") + currentLocation)
```

Comment:
Please dont use translation string with normal string. Use translation parameters instead. Change it everywhere in this file.